cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -74,7 +74,7 @@
       cronjob at:'*/10 * * * *', do:deploy_dir('bin', 'cron', 'delete_twilio_data')
       cronjob at:'* * * * *', do:deploy_dir('bin', 'cron', 'confirm_usage')
       cronjob at:'5 16 * * *', do:deploy_dir('bin', 'cron', 'calculate_workshop_survey_results')
-      cronjob at:'20 6 * * *', do:deploy_dir('bin', 'cron', 'dms_tasks')
+      cronjob at:'20 6 * * 3,6', do:deploy_dir('bin', 'cron', 'dms_tasks')
```

Comment:
That's ok with me since it's only one night a week, and we're on the order of 14-15 hours for a full refresh currently.